content/blog/2016/2016-03-02-gsoc2016-announcement.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Jenkins participates in Google Summer of Code 2016"
+tags:
+- general
+- jenkins2
+- gsoc
+author: oleg_nenashev
+---
+
+We are happy to announce that Jenkins project application has been accepted to [Google Summer of Code 2016](https://developers.google.com/open-source/gsoc/timeline). 
```

Comment:
After "Google Summer of Code" we should introduce the acronym "GSoC", e.g.:

> has been accepted to the Google Summer of Code (GSoC) 2016

That way references to it further down in the document are clearer